Source/WebCore/platform/graphics/egl/GLContext.cpp

```diff
@@ -483,6 +489,36 @@ void GLContext::swapBuffers()
     eglSwapBuffers(m_display.eglDisplay(), m_surface);
 }
 
+void GLContext::swapBuffersWithDamage(const Vector<IntRect>& rects)
+{
+    if (m_type == Surfaceless)
+        return;
+
+    // EGL's swap_buffers_with_damage falls back to full surface damage if not rects are given
+    if (!rects.size() || !m_eglSwapBuffersWithDamage) {
+        swapBuffers();
+        return;
+    }
+
+    auto rectsPtr = WTF::makeUniqueArray<int>(4* rects.size());
```

Comment:
```suggestion
    auto rectsPtr = WTF::makeUniqueArray<int>(4 * rects.size());
```

Reply:
Also, this should use `EGLint`, because it is _not_ guaranteed that it's the same type as `int`. The only guarantee in the EGL specification is that it is an integral type at least 32-bits wide.

 ```suggestion
    auto rectsPtr = WTF::makeUniqueArray<EGLint>(4 * rects.size());
```

Reply:
done